### PR TITLE
Fix suggested project creation

### PR DESCRIPTION
**Pull request by ke4**
ebi-ait/dcp-ingest-central#375

Changes:
- Add basic content when creating a suggested project with `describedBy` and `schema-type` fields

### CODE REVIEW

File: src/test/java/org/humancellatlas/ingest/schemas/SchemaServiceTest.java [hunk 1]
```diff
@@ -28,12 +28,12 @@
     private SchemaScraper schemaScraper;
 
     @Test
-    public void testGetLatestSchemas() {
+    public void testGetLatestSchemasVersions() {
```
Comment (jacobwindsor) [inline src/test/java/org/humancellatlas/ingest/schemas/SchemaServiceTest.java:31]: Thanks for adding tests!

**Review by jacobwindsor — APPROVED** [review]

File: src/test/java/org/humancellatlas/ingest/schemas/SchemaServiceTest.java (hunk 1 above)
Comment (ke4) [reply src/test/java/org/humancellatlas/ingest/schemas/SchemaServiceTest.java:31]: That is the right thing to do, specially if it is easy to do. ;-)

File: src/main/java/org/humancellatlas/ingest/schemas/SchemaService.java [hunk 2]
```diff
@@ -52,10 +52,20 @@
...
+                .collect(Collectors.toList());
+
+        return allLatestSchema.size() > 0 ? allLatestSchema.get(0) : null;
+    }
+
     @Scheduled(fixedDelay = EVERY_24_HOURS)
     public void updateSchemasCollection() {
         String schemaBaseUri = environment.getProperty("SCHEMA_BASE_URI");
 
+        if (schemaBaseUri == null) return;
```
Comment (aaclan-ebi) [inline src/main/java/org/humancellatlas/ingest/schemas/SchemaService.java:67]: I think this env var must always be set. You could raise an error here instead.

**Review by aaclan-ebi — DISMISSED** [review]

File: src/main/java/org/humancellatlas/ingest/schemas/SchemaService.java (hunk 2 above)
Comment (ke4) [reply src/main/java/org/humancellatlas/ingest/schemas/SchemaService.java:67]: I was not sure if I should throw an exception or just silently exit from this scheduled execution.
OK, I am going to throw an exception. Tx for the comment.

Comment (ke4) [reply src/main/java/org/humancellatlas/ingest/schemas/SchemaService.java]: I changed it to throw an exception and also added a test case for it.

Comment (aaclan-ebi) [reply src/main/java/org/humancellatlas/ingest/schemas/SchemaService.java]: cool, thanks!!

**Review by aaclan-ebi — APPROVED** [review]

**Review by jacobwindsor — APPROVED** [review]

**Pull request merged**